packages/@aws-cdk/aws-elasticloadbalancing/test/integ.elb.ts

```diff
@@ -2,6 +2,7 @@
 import * as ec2 from '@aws-cdk/aws-ec2';
 import * as cdk from '@aws-cdk/core';
 import * as elb from '../lib';
+import { InstanceTarget } from '../lib';
```

Comment:
this import isn't necessary since the line above is `import * as elb from '../lib';`. You should just use `elb.InstanceTarget` instead and remove this line.